src/ReactiveUI/Platforms/ios/CommonReactiveSource.cs

```diff
@@ -410,9 +410,13 @@ private static IEnumerable<Update> GetUpdatesForEvent(PendingChange pendingChang
                         .Range(pendingChange.NewStartingIndex, pendingChange.NewItems == null ? 1 : pendingChange.NewItems.Count)
                         .Select(x => Update.CreateAdd(x));
                 case NotifyCollectionChangedAction.Remove:
+                    //Use OldStartingIndex for each "Update.Index" because the batch update processes and removes items sequentially
```

Comment:
Might be worth a space between "//" and the comment here

Reply:
I added these but it doesn't seem to be propagating here...

There's an "Update branch" button that I click and it says

"Update branch attempt failed
Head branch was modified. Review and try the merge again."